tests/approaches/test_nsrt_learning_approach.py

```diff
@@ -84,6 +85,10 @@ def _test_approach(env_name,
                                                    task.goal_holds,
                                                    max_num_steps=CFG.horizon)
             assert task.goal_holds(traj.states[-1])
+    # Do not load NSRTs from pickle file. This is because we cannot load
+    # NSRTs when they are saved as strings (Necessary for behavior)
+    if no_loading:
+        return approach
```

Comment:
why is this returning approach?

Reply:
the function nominally returns approach anyway, this is just a flag that early terminates before we try loading